planetary-social/nos
# #1602: Welcome to Your Feed tip

## Issues covered
#1602 

## Description
Shows the Welcome to Your Feed tip the first time the user lands on the feed. The tip persists on the Feed until the user dismisses it by tapping the `x` button.

## How to test
1. Navigate to the Feed
2. Observe
3. If you dismiss it and want to test again, you can delete the app and reinstall. Or, for developer testing, add this line to `NosApp.init()` after the `Tips.configure()` line:

```
Tips.showAllTipsForTesting()
```

## Screenshots/Video

| Before | After |
|--------|--------|
| ![Simulator Screenshot - iPhone 16 Pro - 2024-10-11 at 16 57 41](https://github.com/user-attachments/assets/55c9f9f9-83a0-4fb8-b213-50bbbc851575) | ![Simulator Screenshot - iPhone 16 Pro - 2024-10-11 at 16 57 24](https://github.com/user-attachments/assets/4fe60e0b-afc4-4b2f-b5b6-0895784e4d9b) |
